user-service/src/contract-tests/resources/application-akkatest.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+akka:
```

Comment:
More of a question than an issue.  I would have expected to see the configuration for testing in a file associated with a test file in the same way that we have a akka profile for production.  Is there a good reason why we don't or it is just convention?

Reply:
I'm not sure I fully understand the comment. This is a test profile as the file name suggests, `akkatest`, the values inside the file are the necessary properties that will override anything within the `akka` profile.